exporter/clickhouseexporter/exporter_logs_test.go

```diff
@@ -121,6 +122,116 @@ func TestExporter_pushLogsData(t *testing.T) {
 	})
 }
 
+func TestLogsClusterConfigOn(t *testing.T) {
+	testClusterConfigOn(t, func(t *testing.T, dsn string, fns ...func(*Config)) {
+		exporter := newTestLogsExporter(t, dsn, fns...)
+		require.NotEmpty(t, exporter.cfg.ClusterClause())
+	})
+}
+
+func TestLogsClusterConfigOff(t *testing.T) {
+	testClusterConfigOff(t, func(t *testing.T, dsn string, fns ...func(*Config)) {
+		exporter := newTestLogsExporter(t, dsn, fns...)
+		require.Empty(t, exporter.cfg.ClusterClause())
+	})
+}
+
+func TestLogsTableEngineConfig(t *testing.T) {
```

Comment:
I suggest using `t.Run` function to group those cases: 
- TestLogsTableEngineConfig
- TestLogsTableEngineConfigWithParams
- TestLogsEmptyTableEngineConfig
- ....

please refer:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/cd25ada2c983be0616c824ba547c01e11cdb3196/exporter/clickhouseexporter/exporter_logs_test.go#L78-L95

Reply:
@Frapschen done.